src/jreleaser/bluesky.tpl

```diff
@@ -0,0 +1,12 @@
+🚀 JReleaser {{projectVersion}} has been released! {{releaseNotesUrl}}
```

Comment:
Linked to previous note. this file is not needed at the moment. Please remove.

Reply:
Gone